pages/Developer/MAP API/map-migration-api.md

```diff
@@ -0,0 +1,164 @@
+
+Try it in our [Public Swagger](https://app.nops.io/public_swagger/)
+
+## Endpoints
+
+### 1. List All Map Migration Projects
+
+- **URL**: `/api/map-migration/`
+- **Method**: `GET`
+- **Description**: Lists all map migration projects for the current client.
+
+#### Request Parameters
+
+None.
+
+#### Response Format
+
+The response is a JSON array containing a list of map migration projects. Each project has the following fields:
+
+- `id`: The ID of the map migration project.
+- `on_prem_non_commercial_db_entity`: Specific incentives related to on-premises non-commercial database.
+- `on_prem_commercial_db_entity`: Specific incentives related to on-premises commercial database.
+- `ec2_non_commercial_db_entity`: Specific incentives related to EC2 non-commercial database.
+- `ec2_commercial_db_entity`: Specific incentives related to EC2 commercial database.
+- `sap_entity`: Specific incentives related to SAP.
+- `baseline_s3_entity`: Specific incentives related to S3.
+- `speciality_services_entity`: Specific incentives related to speciality services.
+- `private_equity_entity`: Specific incentives related toprivate equity.
+- `tagged_resources`: Information about tagged resources, including resource count and cost.
+- `untagged_resources`: Information about untagged resources, including resource count and cost.
+- `credits`: Information about received credits, including records of credit details.
+- `created`: The creation date of the project.
+- `modified`: The last update date of the project.
+- `name`: The name of the project.
+- `migration_id`: The migration ID is identified by the project number in the contract, of the form MPExxxxx (where xxxxx are digits) or a 10-digit identifier such as ABCLMNOPYZ. If you are not sure where to find your MAP Project ID, refer to the top of the first page of your MAP contract.
+- `tag_key`: Must be the exact value string from your AWS contract. This value should be mig followed by the 5 or 10-digit MAP ID. If it’s an older MAP contract, you may also use the server-id (typically d- followed by 9 digits).
+- `server_id`: The server ID, as referred above.
+- `credit_percent`: Prefilled for you to the typical 25%, however you may edit this when creating a project in nOps if your contract terms state a different discount for standard MAP resources.
+- `workload_pk`: Primary key of a workload (deprecated).
+- `date_start`: The start date of the migration (per your MAP contract terms from AWS).
+- `date_end`: The end date of the migration (per your MAP contract terms from AWS).
+- `migration_period`: The migration period in months.
+- `windows_optimization`: Indicates if Windows optimization is enabled.
+- `map_credit_cap`: The MAP credit cap (default is 1.000.000USD).
+- `client`: The ID of the client associated with the project.
```

Comment:
`client`: The client ID associated with the MAP project. Another nOps specific value that gets generated and assigned to each nOps platform client at the time of account creation in nOps. You will find your assigned Client ID at the top right corner of nOps platform screen under your profile name.